src/Cli/dotnet/commands/dotnet-workload/install/IWorkloadManifestUpdater.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Workloads.Workload.Install
+{
+    internal interface IWorkloadManifestUpdater
```

Comment:
This is part of the IAL, does it need to be a separate interface?  Shouldn't these just be methods on `IInstaller`?

Reply:
This interface manages the advertising manifest, so it's not part of the IAL as it's the same across all installation types.